src/unoserver/server.py

```diff
@@ -11,58 +11,62 @@
 
 
 class UnoServer:
-    def __init__(self, interface="127.0.0.1", port="2002"):
+    def __init__(self, interface="127.0.0.1", port="2002", user_installation=None):
         self.interface = interface
         self.port = port
 
+        if user_installation is None:
+            with tempfile.TemporaryDirectory() as tmpuserdir:
```

Comment:
This will delete the temporary directory as soon as the context is exited, so this doesn't work. The easiest way to do this is probably to create it in the main() function instead.

Reply:
Ah I see, will this work [01a1c98](https://github.com/unoconv/unoserver/pull/76/commits/01a1c98bd684dce95e3888f7095875d6d70026e5)?